butterfly-facade/src/test/java/com/paypal/butterfly/facade/ConfigurationTest.java

```diff
@@ -59,6 +63,16 @@ public void testZipOutput(){
         Assert.assertTrue(configuration.isZipOutput());
     }
 
+    /**
```

Comment:
Please, also add a test making sure that, if the constructor with parameters is used, then `isModifyOriginalFolder` returns `false`.